### ISSUE TITLE
Dry/Wet Slider should not be part of the presets

### ISSUE THREAD

**Issue by xherbie** [michaelwillis/dragonfly-reverb#34]
Hi, thanks for that good sounding Reverb. I often use reverbs in a bus (mixbus) and in this case the dry/wet slider is always set to 100% wet. When I change the preset I have to adjust the slider every time. In my opinion the slider should not be part of a preset.

Best regards
Herbert

**Comment by michaelwillis** [comment]
@xherbie Thanks for the feedback. After a year of working on this, I have recently had several people express that they prefer to work with reverb 100% wet on a separate bus from the dry signal. After reading a lot of suggestions of how to accomodate that, I was just thinking the other day that an easy solution would be to make the presets not control the sliders.

I'm still thinking about whether I want to go with this solution. So far it is my favorite, I prefer it to something like adding more controls to the user interface to "lock" the sliders.

**Comment by xherbie** [comment]
@michaelwillis  Yes, excactly what I suggest. Please make the presets not control the sliders for wet/dry :-) 

Best regards

**Comment by jrigg** [comment]
Another vote here for making the wet/dry sliders independent of preset. I also often use reverb 100% wet on a separate bus.

Nice sounding reverb. Thanks for making it available.

**Comment by michaelwillis** [comment]
@xherbie and @jrigg I made a new release in which the presets don't control the sliders:

https://github.com/michaelwillis/dragonfly-reverb/releases/tag/1.0.0-RC2

Let me know what you think.

**Comment by jrigg** [comment]
It works very well. Thanks for making the change.

A minor observation: the .so files have execute permission, which is unnecessary.
